data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/events/EventSubscriber.java

```diff
@@ -68,6 +76,14 @@ public void onEvent(final EventType eventType, final EventSource message)
     }
   }
 
+  private void keepAlive() {
+    final UInt64 timeSeconds = timeProvider.getTimeInSeconds();
+    if (timeSeconds.minusMinZero(lastEventSentSeconds).isGreaterThanOrEqualTo(30)) {
+      sseClient.sendComment(" ");
```

Comment:
We shouldn't try to send the comment on the same thread that calls `onEvent` otherwise if the client is slow reading we might wind up blocking important threads.  Need to use `asyncRunner.runAsync` for this (but probably still update lastEventSentSeconds immediately to avoid scheduling sending multiple keep alives).